### PR TITLE
Improve docstring comments

### PR DESCRIPTION
**Pull request by mattsta**
Only docstring formatting changes.

Just replaces three line docstrings with one line docstrings.

Half the project was using proper format and the other half a "too many
newlines" format, so normalize the too-many-newlines to standard
formatting.

### CODE REVIEW

File: src/dispatch/event/service.py [hunk 1]
```diff
@@ -62,9 +52,7 @@ def update(*, db_session, event: Event, event_in: EventUpdate) -> Event:
 
 
 def delete(*, db_session, event_id: int):
-    """
-    Deletes an event
-    """
+    """Deletes an event"""
```
Comment (mvilanova) [inline src/dispatch/event/service.py:55]:
```suggestion
    """Deletes an event."""
```

File: src/dispatch/event/service.py [hunk 2]
```diff
@@ -80,9 +68,7 @@ def log(
     ended_at: datetime = None,
     details: dict = None,
 ) -> Event:
-    """
-    Logs an event
-    """
+    """Logs an event"""
```
Comment (mvilanova) [inline src/dispatch/event/service.py:71]:
```suggestion
    """Logs an event."""
```

File: src/dispatch/incident_cost/service.py [hunk 3]
```diff
@@ -21,25 +21,19 @@
...
-    """
+    """Gets an incident cost by its id."""
     return db_session.query(IncidentCost).filter(IncidentCost.id == incident_cost_id).one_or_none()
 
 
 def get_by_incident_id(*, db_session, incident_id: int) -> List[Optional[IncidentCost]]:
-    """
-    Gets incident costs by incident id.
-    """
+    """Gets incident costs by incident id."""
```
Comment (mvilanova) [inline src/dispatch/incident_cost/service.py:29]:
```suggestion
    """Gets incident costs by their incident id."""
```

File: src/dispatch/incident_cost/service.py [hunk 4]
```diff
@@ -21,25 +21,19 @@
...
     return db_session.query(IncidentCost).filter(IncidentCost.incident_id == incident_id)
 
 
 def get_by_incident_id_and_incident_cost_type_id(
     *, db_session, incident_id: int, incident_cost_type_id: int
 ) -> List[Optional[IncidentCost]]:
-    """
-    Gets incident costs by incident id and incident cost type id.
-    """
+    """Gets incident costs by incident id and incident cost type id."""
```
Comment (mvilanova) [inline src/dispatch/incident_cost/service.py:36]:
```suggestion
    """Gets incident costs by their incident id and incident cost type id."""
```

File: src/dispatch/incident_cost/views.py [hunk 5]
```diff
@@ -19,17 +19,13 @@
...
+    """Get all incident costs, or only those matching a given search term."""
     return search_filter_sort_paginate(model="IncidentCost", **common)
 
 
 @router.get("/{incident_cost_id}", response_model=IncidentCostRead)
 def get_incident_cost(*, db_session: Session = Depends(get_db), incident_cost_id: int):
-    """
-    Get an incident cost by id.
-    """
+    """Get an incident cost by id."""
```
Comment (mvilanova) [inline src/dispatch/incident_cost/views.py:28]:
```suggestion
    """Get an incident cost by its id."""
```

File: src/dispatch/incident_cost/views.py [hunk 6]
```diff
@@ -62,9 +56,7 @@ def update_incident_cost(
     incident_cost_id: int,
     incident_cost_in: IncidentCostUpdate,
 ):
-    """
-    Update an incident cost by id.
-    """
+    """Update an incident cost by id."""
```
Comment (mvilanova) [inline src/dispatch/incident_cost/views.py:59]:
```suggestion
    """Update an incident cost by its id."""
```

File: src/dispatch/incident_cost_type/views.py [hunk 7]
```diff
@@ -19,17 +19,13 @@
...
+    """Get all incident cost types, or only those matching a given search term."""
     return search_filter_sort_paginate(model="IncidentCostType", **common)
 
 
 @router.get("/{incident_cost_type_id}", response_model=IncidentCostTypeRead)
 def get_incident_cost_type(*, db_session: Session = Depends(get_db), incident_cost_type_id: int):
-    """
-    Get an incident cost type by id.
-    """
+    """Get an incident cost type by id."""
```
Comment (mvilanova) [inline src/dispatch/incident_cost_type/views.py:28]:
```suggestion
    """Get an incident cost type by its id."""
```

File: src/dispatch/incident_cost_type/views.py [hunk 8]
```diff
@@ -64,9 +58,7 @@ def update_incident_cost_type(
     incident_cost_type_id: int,
     incident_cost_type_in: IncidentCostTypeUpdate,
 ):
-    """
-    Update an incident cost type by id.
-    """
+    """Update an incident cost type by id."""
```
Comment (mvilanova) [inline src/dispatch/incident_cost_type/views.py:61]:
```suggestion
    """Update an incident cost type by its id."""
```

File: src/dispatch/individual/views.py [hunk 9]
```diff
@@ -42,9 +38,7 @@ def create_individual(
 
 @router.get("/{individual_contact_id}", response_model=IndividualContactRead)
 def get_individual(*, db_session: Session = Depends(get_db), individual_contact_id: int):
-    """
-    Get a individual contact.
-    """
+    """Get a individual contact."""
```
Comment (mvilanova) [inline src/dispatch/individual/views.py:41]:
```suggestion
    """Get an individual contact."""
```

File: src/dispatch/individual/views.py [hunk 10]
```diff
@@ -63,9 +57,7 @@ def update_individual(
     individual_contact_id: int,
     individual_contact_in: IndividualContactUpdate,
 ):
-    """
-    Update a individual contact.
-    """
+    """Update a individual contact."""
```
Comment (mvilanova) [inline src/dispatch/individual/views.py:60]:
```suggestion
    """Update an individual contact."""
```

File: src/dispatch/individual/views.py [hunk 11]
```diff
@@ -83,9 +75,7 @@ def update_individual(
     dependencies=[Depends(PermissionsDependency([SensitiveProjectActionPermission]))],
 )
 async def delete_individual(*, db_session: Session = Depends(get_db), individual_contact_id: int):
-    """
-    Delete a individual contact.
-    """
+    """Delete a individual contact."""
```
Comment (mvilanova) [inline src/dispatch/individual/views.py:78]:
```suggestion
    """Delete an individual contact."""
```

File: src/dispatch/notification/views.py [hunk 12]
```diff
@@ -19,17 +19,13 @@
...
+    """Get all notifications, or only those matching a given search term."""
     return search_filter_sort_paginate(model="Notification", **common)
 
 
 @router.get("/{notification_id}", response_model=NotificationRead)
 def get_notification(*, db_session: Session = Depends(get_db), notification_id: int):
-    """
-    Get a notification by id.
-    """
+    """Get a notification by id."""
```
Comment (mvilanova) [inline src/dispatch/notification/views.py:28]:
```suggestion
    """Get a notification by its id."""
```

File: src/dispatch/notification/views.py [hunk 13]
```diff
@@ -62,9 +56,7 @@ def update_notification(
     notification_id: int,
     notification_in: NotificationUpdate,
 ):
-    """
-    Update a notification by id.
-    """
+    """Update a notification by id."""
```
Comment (mvilanova) [inline src/dispatch/notification/views.py:59]:
```suggestion
    """Update a notification by its id."""
```

File: src/dispatch/organization/views.py [hunk 14]
```diff
@@ -62,9 +58,7 @@ def create_organization(
 
 @router.get("/{organization_id}", response_model=OrganizationRead)
 def get_organization(*, db_session: Session = Depends(get_db), organization_id: int):
-    """
-    Get a organization.
-    """
+    """Get a organization."""
```
Comment (mvilanova) [inline src/dispatch/organization/views.py:61]:
```suggestion
    """Get an organization."""
```

File: src/dispatch/organization/views.py [hunk 15]
```diff
@@ -82,9 +76,7 @@ def update_organization(
     organization_id: int,
     organization_in: OrganizationUpdate,
 ):
-    """
-    Update a organization.
-    """
+    """Update a organization."""
```
Comment (mvilanova) [inline src/dispatch/organization/views.py:79]:
```suggestion
    """Update an organization."""
```

File: src/dispatch/participant/service.py [hunk 16]
```diff
@@ -15,18 +15,14 @@
 
 
 def get(*, db_session, participant_id: int) -> Optional[Participant]:
-    """
-    Get a participant by id.
-    """
+    """Get a participant by id."""
```
Comment (mvilanova) [inline src/dispatch/participant/service.py:18]:
```suggestion
    """Get a participant by its id."""
```

File: src/dispatch/tag/views.py [hunk 17]
```diff
@@ -37,18 +33,14 @@ def get_tag(*, db_session: Session = Depends(get_db), tag_id: str):
...
     tag = create(db_session=db_session, tag_in=tag_in)
     return tag
 
 
 @router.put("/{tag_id}", response_model=TagRead)
 def update_tag(*, db_session: Session = Depends(get_db), tag_id: int, tag_in: TagUpdate):
-    """
-    Given its unique ID, update details of an tag.
-    """
+    """Given its unique ID, update details of an tag."""
```
Comment (mvilanova) [inline src/dispatch/tag/views.py:43]:
```suggestion
    """Update a tag."""
```

File: src/dispatch/tag/views.py [hunk 18]
```diff
@@ -58,9 +50,7 @@ def update_tag(*, db_session: Session = Depends(get_db), tag_id: int, tag_in: Ta
 
 @router.delete("/{tag_id}")
 def delete_tag(*, db_session: Session = Depends(get_db), tag_id: int):
-    """
-    Delete an tag, returning only an HTTP 200 OK if successful.
-    """
+    """Delete an tag, returning only an HTTP 200 OK if successful."""
```
Comment (mvilanova) [inline src/dispatch/tag/views.py:53]:
```suggestion
    """Delete a tag, returning only an HTTP 200 OK if successful."""
```

File: src/dispatch/term/views.py [hunk 19]
```diff
@@ -57,9 +49,7 @@ def update_term(*, db_session: Session = Depends(get_db), term_id: int, term_in:
 
 @router.delete("/{term_id}", response_model=TermRead)
 def delete_term(*, db_session: Session = Depends(get_db), term_id: int):
-    """
-    Update a term.
-    """
+    """Update a term."""
```
Comment (mvilanova) [inline src/dispatch/term/views.py:52]:
```suggestion
    """Delete a term."""
```

File: src/dispatch/term/views.py [hunk 20]
```diff
@@ -34,9 +30,7 @@ def create_term(*, db_session: Session = Depends(get_db), term_in: TermCreate):
 
 @router.get("/{term_id}", response_model=TermRead)
 def get_term(*, db_session: Session = Depends(get_db), term_id: int):
-    """
-    Update a term.
-    """
+    """Update a term."""
```
Comment (mvilanova) [inline src/dispatch/term/views.py:33]:
```suggestion
    """Get a term."""
```

**Review by mvilanova — CHANGES_REQUESTED** [review]
Thanks for the PR. Added some code change suggestions.

**Comment by mattsta** [comment]
oh, sure, looks great. 👍

My change had just compressed the docstrings down to one line without reading the context because scanning through files looked a bit off.

**Comment by mvilanova** [comment]
Cool, thanks again!

**Pull request merged**